fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -2585,14 +2593,21 @@ public Builder setMutualIndexing() {
              * @return this builder
              */
             @API(API.Status.EXPERIMENTAL)
-            public Builder setMutualIndexing(final boolean useMutualIndexing) {
+            public Builder setMutualIndexing(boolean useMutualIndexing) {
                 this.useMutualIndexing = useMutualIndexing;
                 if (!useMutualIndexing) {
                     useMutualIndexingBoundaries = null;
                 }
                 return this;
             }
 
+            @API(API.Status.EXPERIMENTAL)
+            public Builder setMutualIndexing(boolean useMutualIndexing, boolean mainIndexer) {
```

Comment:
This could use javadoc, particularly pointing out that having more than one worker where `mainIndexer==true` running at the same time could result in a lot of conflicts, and cause the index build to take much longer. But if all main indexers fail, the index might not be built completely.

Plus, all the javadoc from the other `setMutualIndexing`

Reply:
✅